libraries/heft-config-file/src/ConfigurationFile.ts

```diff
@@ -55,7 +55,12 @@ export enum PathResolutionMethod {
    * Treat the property as a NodeJS-style require/import reference and resolve using standard
    * NodeJS filesystem resolution
    */
-  NodeResolve
+  NodeResolve,
```

Comment:
Wonder why I capitalized this.

Reply:
Enum members are conventionally capitalized.  I almost implemented an ESLint rule to enforce that, but IIRC it would break a lot of legacy code.

Reply:
Maybe we should make a lint rule.

Reply:
It's probably fine to rename it and make a note in the changelog.

Reply:
Let's handle it in a separate PR.

My vote would be do one big PR that renames all the Heft API enums to use `InitialCaps`.